### DISCUSSION TITLE
Tags for Post?

### DISCUSSION THREAD

**Discussion by cybertronbaka** [jigmeloday/blog-post-api#16] [Q&A] [answered]
We could think of adding tags to post. If so, how should we do that?

**Comment by jigmeloday** [comment, marked as answer]
yes we could do something like this,

In the ERD, include a new table called `tags` to represent the categories or tags associated with articles. The `Tags` table should have attributes such as `tag_id` `(primary key)` and `name.` Establish a many-to-many relationship between articles and tags by creating an "ArticleTags" table. The `ArticleTags` table should contain columns like `article_id` and `tag_id,` which act as foreign keys referencing the primary keys of the `Articles` and `Tags` tables, respectively. This relationship allows an article to have multiple tags, while a tag can belong to multiple articles.
